operations/app/terraform/modules/function_app/apps.tf

```diff
@@ -40,5 +40,15 @@ module "pdi_function_app" {
     "${each.value.mi_accountName}"        = each.value.mi_accountValue
     fhir_url                              = each.value.fhir_url
     "AzureWebJobs.convertToFhir.Disabled" = each.value.AzureWebJobs_convertToFhir_Disabled
+    SMARTYSTREETS_AUTH_ID                 = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/SmartyStreetsAuthID)"
+    SMARTYSTREETS_AUTH_TOKEN              = "@Microsoft.KeyVault(SecretUri=https://${var.resource_prefix}-app-kv.vault.azure.net/secrets/SmartyStreetsAuthToken)"
```

Comment:
These are currently configured in Azure as the literal values.  Have these values been loaded in to the key vault?  If so, I think Azure should be updated, and this tf representation can stay as-is.

Reply:
that's the idea - I added entries to keyvault for both envs